pkg/controller/webhook.go

```diff
@@ -124,7 +131,7 @@ func CallEventWebhook(r *flaggerv1.Canary, w flaggerv1.CanaryWebhook, message, e
 			payload.Metadata[key] = value
 		}
 	}
-	return callWebhook(w.URL, payload, "5s")
+	return callWebhook(w.URL, payload, "5s", 0)
```

Comment:
is there a particular reason why we are specifying the number of retries as 0 instead of `w.Retries` here?